### DISCUSSION TITLE
Primer commit para el tp

### DISCUSSION THREAD

**Discussion by PabloBarrionuevo** [AstorArico72/Proyecto-Transversal-1#5] [Ideas]
Se que aun no lo solicitan, pero por temas del trabajo como no estoy muy atento a los horarios de clases, propongo:
Astor, (para mas adelante) hacer un primer commit con la estructura del proyecto (de la carpeta src, aunque este vacía) para que nosotros vinculemos un nuevo proyecto y solicitemos (mediante pull) la estructura, luego dividimos las tareas para todos tener los mismos archivos.

Ideas para el proyecto en la carpeta src: hacer 3 paquetes, (1) para los recursos y el main, (2) para la parte visual, (3) para la conexión con la BD; así podemos trabajar la parte (1) de los recursos con la parte visual (2) y con lo que la BD devuelva en memoria (3).

(1) En los recursos obviamente tendríamos a las clases: Alumno, Materia, Cursada, Cualquier otro que sea necesario. (según lo que se ve en la grabación de las clases) Además una Clase Universidad para tener los métodos de las funcionalidades (que liste alumnos por materia, etc.) y el Main.

(2) En la parte visual lo vemos mas adelante.

(3) En la parte de conexión al menos una clase que sirva para conectarse (y des-conectarse) y para hacer CRUD con la BD (Crear, leer, actualizar o eliminar algo de ella).

Quedando algo así:
Desde la parte visual por ejemplo, si solicitan armar un listado de alumnos por materia, entonces la clase Universidad pida los datos a la BD, la almacene en una colección apropiada y muestre el listado en la parte visual (usando las clases de Alumno, Materia, Cursada según corresponda). La Universidad se encargaría de hacer todo el trabajo.

Cuando sea necesario podemos organizar un Zoom, para ver esto rápido, mis horarios en los que puedo, son de 22:00 hs. en adelante durante días de semana laborales, y fines de semana si no me mandan a trabajar en todo el día.

**Comment by Gianibruno** [comment]
Astor, estoy de acuerdo con Pablo, por favor hacé un primer commit de un proyecto, para que nosotros nos podamos vincular a el de forma facil e ir pudiendo ver el avance de cada uno de una manera sencilla y unificada.
